src/Sulu/Bundle/AdminBundle/Command/DownloadLanguageCommand.php

```diff
@@ -75,23 +118,39 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $translations = [];
         }
 
+        $packages = [];
         foreach ($suluPackages as $suluPackage) {
-            $translations = array_merge($translations, $this->downloadLanguage($output, $language, $suluPackage));
+            $packageTranslations = $this->downloadLanguage($output, $language, $suluPackage, $translationBaseUrl);
+            $packages[$suluPackage] = [
+                'Package' => $suluPackage,
+                'Translations' => count($packageTranslations),
+            ];
+            $translations = array_merge($translations, $packageTranslations);
+        }
+
+        $ui->newLine();
+
+        if (empty($translations)) {
+            $ui->warning(sprintf('No translations found for language %s.', $language));
```

Comment:
`Translation for language "%s" does not exist yet. Reach out to the Sulu team via https://sulu.io/contact-us if you are interested in creating it. Your contribution is highly appreciated!`